Sprint-2/implement/querystring.js

```diff
@@ -6,11 +6,11 @@ function parseQueryString(queryString) {
   const keyValuePairs = queryString.split("&");
 
   for (const pair of keyValuePairs) {
-    const [key, value] = pair.split("=");
-    queryParams[key] = value;
+    //const [key, value] = pair.split("=", 2);
+     const [key, value] = [pair.slice(0,pair.indexOf('=')),pair.slice(pair.indexOf('=')+1)];
```

Comment:
I find this line a bit hard to understand. Maybe we could split it up into multiple variables?

```suggestion
const equalsIndex = pair.indexOf('=');
     const [key, value] = [pair.slice(0,equalsIndex),pair.slice(pair.indexOf('=')+1)];
```

Breakingn it up into multiple lines with clear names makes it easier to follow